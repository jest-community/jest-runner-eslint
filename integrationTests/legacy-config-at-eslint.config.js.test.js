const { version } = require('eslint/package.json');
const semver = require('semver');
const runJest = require('./runJest');

(semver.satisfies(version, '<8.41') ? it : it.skip)(
  "Does not try to use flat config on eslint versions that don't support it",
  async () => {
    expect(
      await runJest('legacy-config-at-eslint.config.js'),
    ).toMatchSnapshot();
  },
);
