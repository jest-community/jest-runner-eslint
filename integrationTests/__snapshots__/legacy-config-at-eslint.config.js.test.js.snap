// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Does not try to use flat config on eslint versions that don't support it 1`] = `
"PASS __eslint__/file.js
  ● Console

  console.warn
    
    /mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/legacy-config-at-eslint.config.js/__eslint__/file.js
      3:1  warning  Unexpected console statement  no-console
    
    ✖ 1 problem (0 errors, 1 warning)

  ✓ ESLint 

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;
